src/services/JobCandidateService.js

```diff
@@ -203,8 +203,8 @@ fullyUpdateJobCandidate.schema = Joi.object().keys({
     userId: Joi.string().uuid().required(),
     status: Joi.jobCandidateStatus().default('open'),
     externalId: Joi.string().allow(null).default(null),
-    resume: Joi.string().uri().allow(null).default(null),
-    remark: Joi.string().allow(null).default(null)
+    resume: Joi.string().uri().allow('').allow(null).default(null),
+    remark: Joi.string().allow('').allow(null).default(null)
```

Comment:
@sushilshinde we also have a reusable rule `stringAllowEmpty` https://github.com/topcoder-platform/taas-apis/blob/dev/src/bootstrap.js#L26 which could be reused here for consistency.